code/controllers/configuration/entries/game_options.dm

```diff
@@ -394,3 +394,8 @@
 /datum/config_entry/flag/sdql_spells
 
 /datum/config_entry/flag/native_fov
+
+/datum/config_entry/number/station_goal_budget
+	default = 1
+	min_val = 0
+	integer = FALSE
```

Comment:
?

Reply:
This determines how many station goals are set per shift.  The default is one station goal but as a config option it can be changed to multiple or none depending on what server hosts want.

Reply:
But why is it not an integer

Reply:
because its a float

Reply:
Aren't all our weights ints?

Reply:
Pretty sure the weighted value can be a fraction.

https://github.com/tgstation/tgstation/blob/02031bc9b4076c61779925a41932239170c8f0d9/code/modules/station_goals/station_goal.dm#L4-L9

Reply:
I see, it's not using pickweight.